README.md

```diff
@@ -129,22 +129,29 @@ add_filter( 'wp_sentry_dsn', 'customize_sentry_dsn' );
 
 #### `wp_sentry_options` (array)
 
-You can use this filter to customize the Sentry options used to initialize the PHP tracker.
+You can use this filter to customize the [Sentry options](https://docs.sentry.io/error-reporting/configuration/) used to initialize the PHP tracker.
 
 Example usage:
 
 ```php
 /**
  * Customize sentry options.
+ * 
+ * Note: Items prefixed with `regex:` in blacklistUrls and whitelistUrls option arrays
+ * will be translated into pure RegExp.
  *
  * @param array $options The current sentry options.
  *
  * @return array
+ * 
+ * @link https://docs.sentry.io/error-reporting/configuration/
  */
 function customize_sentry_options( array $options ) {
     return array_merge( $options, array(
-        'tags' => array(
-            'my-custom-tag' => 'custom value',
```

Comment:
Just a small notice: I removed `tags` array because now it supposed to be moved into a `context` sub-array.
But I think adding a clear separation between options and context is better, and even promoted by new versions of SDKs (for JS v4 and for PHP v2).